packages/jest-config/src/normalize.js

```diff
@@ -64,15 +64,23 @@ const setupPreset = (
   let preset;
   const presetPath = replaceRootDirInPath(options.rootDir, optionsPreset);
   const presetModule = Resolver.findNodeModule(
-    presetPath.endsWith(JSON_EXTENSION)
+    presetPath.charAt(0) === '.'
```

Comment:
You can do `[0]` instead of `charAt`. We aren't running this code in IE6.

Reply:
IE6 <3

You can also do `.startsWith('.')`